Pkgfile

```diff
@@ -5,7 +5,7 @@
 format: v1alpha2
```

Comment:
https://github.com/dsseng/talos-tools/blob/7ae8e50bdbbc32281f3ff09d5b6512ede6639fd9/Pkgfile#L1 this also needs updating